modules/acm_sku/acm_sku.info.yml

```diff
@@ -5,11 +5,11 @@ core: 8.x
 package: Commerce
 dependencies:
   - acquia_commercemanager:acm
+  - acquia_commercemanager:acm_sku_stock
   - key_value_field:key_value_field
   - inline_entity_form:inline_entity_form
   - drupal:menu_ui
   - drupal:node
   - drupal:path
-  - pcb:pcb
```

Comment:
It should be removed from composer as well

Reply:
removed

Reply:
adding back, it is now required by another cache `cache.commerce_user`